final_prep/mod_zero_hero.rb

```diff
@@ -2,51 +2,93 @@
 
 # Declare two variables - hero_name AND special_ability - set to strings
 
+hero_name = "Captain Underpants"
+special_ability = "charisma"
+
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
+greeting = "Hello, I am #{hero_name}."
+catchphrase = "I have the ability of #{special_ability}."
+
 # Declare two variables - power AND energy - set to integers
 
+power = 10
+energy = 150
+
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
 
+full_power = power * 500
+full_energy = energy * 150
+
 # Declare two variables - is_human and identity_concealed - assigned to booleans
 
+is_human = true
+identity_concealed = true
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
 
+arch_enemies = ["Miss Diapers", "Captain Overpants", "Clogged Gutters"]
+sidekick = ["Doughnuts", "A Cat", "Superman"]
+
 # Print the first sidekick to your terminal
 
+p sidekick[0]
+
 # Print the last arch_enemy to the terminal
 
+p arch_enemies[2]
```

Comment:
Can you think of an alternative way to print the last `arch_enemy` to the terminal, if you didn't know the length of the array?